specifications/api/swagger.yaml

```diff
@@ -4124,6 +4124,17 @@ x-errMalformedTxPayload: &errMalformedTxPayload
       type: string
       enum: ['malformed_tx_payload']
 
+x-errMempoolIsFull: &errMempoolIsFull
+  <<: *responsesErr
+  title: malformed_tx_payload
```

Comment:
This doesn't look right to me: the payload isn't malformed in this case.

Reply:
copy and paste is not bullet-proof :-/